README.md

```diff
@@ -1,11 +1,75 @@
-# Northcoders News API
+# Northcoders Solo Project News API
 
-## Background
+## Hosted version
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+Here is the link to the hosted version of the project:\
+[https://nc-news-solo-project.herokuapp.com](https://nc-news-solo-project.herokuapp.com)
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+## Summary
 
-## Setup
+I build this API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture. The database contains some news, topics, comments which can be accessed by a frontend web application by using the APIs.
 
-### Connect to the databases locally
+## Instructions
+
+### 1. Clone this repository
+
+### 2. You will need to install these dependencies:
+
+```
+jest:
+npm install --save-dev jest
+
+jest-sorted:
+npm install --save-dev jest-sorted
+
+pg-format:
+npm install pg-format
+
+supertest:
+npm install supertest --save-dev
+
+express:
+npm install express
+
+dotenv:
+npm install dotenv --save
+
+pg:
+npm install pg
+```
```

Comment:
You can condense this into : `
npm install
npm install -D jest
npm install -D jest-sorted
npm install -D supertest
`
Just so it doesn't look so busy!